src/Domain/Service/SubscriptionsTrackingService.php

```diff
@@ -78,8 +78,10 @@ public function refreshSubscriptions(string $extensionId, array $contributorIds)
                 $subscription->confirm();
             } else {
                 $extension = $this->extensionRepository->findOrCreate($extensionId);
+                $extension->confirm();
```

Comment:
I would rather replace 
`$this->extensionRepository->find($extensionId)` line 54
with 
`$this->extensionRepository->findOrCreate($extensionId)`
and remove these 2 lines to keep extension handling in one place, don’t you think ?

Reply:
@JalilArfaoui done